### PR TITLE
data: dirwolf

### PR DESCRIPTION
**Pull request by dirwolf**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/dirwolf.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Dhruv rawat",
+  "bio": "Computer science undergrad",
+  "type": "personal"
```
Comment (sital002) [inline data/dirwolf.json:4]:
```suggestion
```
type property isnot required

File: data/dirwolf.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "avatar": "https://avatars.githubusercontent.com/u/92151410?v=4"
```
Comment (sital002) [inline data/dirwolf.json:5]:
```suggestion
```
avatar property isnot required anymore

File: data/dirwolf.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    {
+      "name": "Youtube",
+      "url": "https://www.youtube.com/channel/UCuLxLMEyO8NambbHWwA5rew",
+      "icon": "FaYoutube"
+    },
+      {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/_dhruv.rawat_/",
+      "icon": "FaInstagram"
+    },
```
Comment (sital002) [inline data/dirwolf.json:21]:
```suggestion
    }
```
The trailing comma isnot required.

**Review by sital002 — CHANGES_REQUESTED** [review]
Please make the necessary changes to get your profile merged.

**Review by SaraJaoude — APPROVED** [review]
I have committed the changes to make JSON pass and  I can now merge your PR.

**Pull request merged**